examples/workflow/3_perform_fit.ipynb

```diff
@@ -115,10 +115,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from copy import deepcopy\n",
-    "\n",
-    "initial_parameters = deepcopy(estimator.parameters)\n",
-    "initial_parameters"
+    "estimator.parameters"
```

Comment:
i meant like `initial_parameters = estimator.parameters`
and then just following the code